main.go

```diff
@@ -0,0 +1,195 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/drone/drone-go/drone"
+	"github.com/drone/drone-go/plugin"
+)
+
+var (
+	build     string
+	buildDate string
+)
+
+func main() {
+	fmt.Printf("Drone ECS Plugin built at %s\n", buildDate)
+
+	workspace := drone.Workspace{}
+	repo := drone.Repo{}
+	build := drone.Build{}
+	vargs := Params{}
+
+	plugin.Param("workspace", &workspace)
+	plugin.Param("repo", &repo)
+	plugin.Param("build", &build)
+	plugin.Param("vargs", &vargs)
+	plugin.MustParse()
+
+	if len(vargs.AccessKeyID) == 0 {
+		fmt.Println("Please provide an access key")
+
+		os.Exit(1)
+		return
+	}
+
+	if len(vargs.SecretAccessKey) == 0 {

```

Comment:
Can you use the naming like aws-codedeploy please? The JSON naming should be similar for the AWS components :)